cashctrl_ledger/ledger.py

```diff
@@ -197,6 +203,36 @@ def settings_clear(self):
     # ----------------------------------------------------------------------
     # Accounts
 
+    def sanitize_account_groups(self, groups: pd.Series) -> pd.Series:
+        """Ensure account group paths are sanitized with a leading slash and a valid root node.
```

Comment:
Proposal to clarify:
```
Ensure account groups start with a leading slash and a valid root node.

Account categories in CashCtrl must be assigned to a pre-defined root node. 
Additional root categories are not allowed, and the five pre-defined root categories can not be deleted.
```
instead of:
```
Ensure account group paths are sanitized with a leading slash and a valid root node.
```